### DISCUSSION TITLE
Applying for student pack but couldn't select my school from the map..

### DISCUSSION THREAD

**Discussion by Lucien2714** [community/community#31973] [GitHub Education]
I am currently a highschool student studying in mainland China, I wasn't able to select my school in the map (I tried searching the school name but it had no result.) Any suggestion on what I should do?

**Comment by 18jplin** [comment]
come on 的喂

**Comment by BLKKKBVSIK** [comment]
You can usually add your own school because GitHub isn't aware of all schools.
You will have to confirm your student email address later on.

**Reply by Lucien2714** [reply]
Thanks for the reply! Well I didn't find the entrance to "adding new schools" but I managed to select my school in the map eventually by typing the full address of my school lol hope it work!
